contrib/windows-cmake/CMakeLists.txt

```diff
@@ -64,11 +121,30 @@ add_library(libhwloc
     ${TOPDIR}/hwloc/topology-xml-nolibxml.c
     ${TOPDIR}/hwloc/topology-windows.c
     ${TOPDIR}/hwloc/topology-x86.c
+    $<$<BOOL:${HWLOC_HAVE_LIBXML2}>:${TOPDIR}/hwloc/topology-xml-libxml.c>
+    $<$<BOOL:${HWLOC_HAVE_OPENCL}>:${TOPDIR}/hwloc/topology-opencl.c>
+    $<$<BOOL:${HAVE_CUDA}>:${TOPDIR}/hwloc/topology-cuda.c>
+)
+
+target_link_libraries(hwloc PRIVATE
+    $<$<BOOL:${HWLOC_HAVE_LIBXML2}>:LibXml2::LibXml2>
+    $<$<BOOL:${HWLOC_HAVE_OPENCL}>:OpenCL::OpenCL>
```

Comment:
are those variables exported in the CMake -config.cmake? Otherwise this will always evaluate to false in user projects.

Reply:
currently there aren't any CMake package scripts installed. This is something simple to add--to make Hwloc install CMake packaging information instead of simply the binary artifacts as it does now.

Reply:
hmm the CMakeLists.txt also needs to generate a pkg-config file if it is not already doing it. Otherwise usage via meson/autotools is difficult.

Reply:
brice says to put in another PR